apps/frontend/src/pages/dashboard/libs/components/user-history.tsx

```diff
@@ -0,0 +1,30 @@
+import { Loader } from "~/libs/components/components.js";
+import { DataStatus } from "~/libs/enums/enums.js";
+import { useAppSelector } from "~/libs/hooks/hooks.js";
+
+import styles from "./styles.module.css";
+import { UserHistoryCard } from "./user-history-card.js";
```

Comment:
Maybe it can be done like this?
```suggestion
const Card = ({ route }: CardProperties): React.JSX.Element => {
	const routeLine = { geometry: route.actualGeometry, id: "actual" };

	return (
		<Link to={configureString(AppRoute.ROUTES, { id: route.id.toString() })}>
			<div className={styles["route-container"]}>
				<h1 className={styles["label"]}>{route.routeName}</h1>
				<MapProvider routeLine={routeLine} />
			</div>
		</Link>
	);
};
```